index.html

```diff
@@ -0,0 +1,37 @@
+
+
+<!DOCTYPE html>
+<html>
+<head>
+   <title>Grow A Tree</title>
+</head>
+<body>
+   <h1>Trees Are Great.</h1>
+   <form action="#" method="post">
```

Comment:
This form is competing with your event handler for `Enter` button press. We want to see you handle the event rather than letting the form do it.